store/sqlstore/link_metadata_store.go

```diff
@@ -53,15 +55,16 @@ func (s SqlLinkMetadataStore) Save(metadata *model.LinkMetadata) (*model.LinkMet
 
 func (s SqlLinkMetadataStore) Get(url string, timestamp int64) (*model.LinkMetadata, *model.AppError) {
 	var metadata *model.LinkMetadata
-
-	err := s.GetReplica().SelectOne(&metadata,
-		`SELECT
-			*
-		FROM
-			LinkMetadata
-		WHERE
-			URL = :URL
-			AND Timestamp = :Timestamp`, map[string]interface{}{"URL": url, "Timestamp": timestamp})
+	query := s.getQueryBuilder().
+		Select("*").
+		From("LinkMetadata").
+		Where(sq.Eq{"URL": url}).
+		Where(sq.Eq{"Timestamp": timestamp})
```

Comment:
These two can be merged in a single call to where:

```suggestion
		Where(sq.Eq{"URL": url, "Timestamp": timestamp})
```